tests/template_tests/syntax_tests/test_i18n.py

```diff
@@ -401,18 +410,23 @@ def test_i18n37(self):
     # Test whitespace in filter arguments
     @setup({'i18n38': '{% load i18n custom %}'
                       '{% get_language_info for "de"|noop:"x y" as l %}'
-                      '{{ l.code }}: {{ l.name }}/{{ l.name_local }} bidi={{ l.bidi }}'})
+                      '{{ l.code }}: {{ l.name }}/{{ l.name_local }}/'
+                      '{{ l.name_translated }} bidi={{ l.bidi }}'})
     def test_i18n38(self):
-        output = self.engine.render_to_string('i18n38')
-        self.assertEqual(output, 'de: German/Deutsch bidi=False')
+        with translation.override('cs'):
+            output = self.engine.render_to_string('i18n38')
+        self.assertEqual(output, 'de: German/Deutsch/německy bidi=False')
 
     @setup({'i18n38_2': '{% load i18n custom %}'
                         '{% get_language_info_list for langcodes|noop:"x y" as langs %}'
                         '{% for l in langs %}{{ l.code }}: {{ l.name }}/'
-                        '{{ l.name_local }} bidi={{ l.bidi }}; {% endfor %}'})
+                        '{{ l.name_local }}/{{ l.name_translated }} '
+                        'bidi={{ l.bidi }}; {% endfor %}'})
     def test_i18n38_2(self):
-        output = self.engine.render_to_string('i18n38_2', {'langcodes': ['it', 'no']})
-        self.assertEqual(output, 'it: Italian/italiano bidi=False; no: Norwegian/norsk bidi=False; ')
+        with translation.override('cs'):
+            output = self.engine.render_to_string('i18n38_2', {'langcodes': ['it', 'fr']})
+        self.assertEqual(output, 'it: Italian/italiano/italsky bidi=False; '
```

Comment:
prefer if you use hanging indent style for this assertion to match the other tests